pkg/chartutil/testdata/subpop/charts/subchart1/values.yaml

```diff
@@ -23,7 +23,7 @@ overridden-chartA:
   SCAbool: true
   SCAfloat: 3.14
   SCAint: 100
-  SCAstring: "jabathehut"
+  SCAstring: "jabbathehut"
```

Comment:
I'm assuming this is a typo and not something intentional.

I generally hope that places which intentionally misspell things include very clear language indicating why they're misspelling things.